SCRIPTS.md

```diff
@@ -54,6 +54,12 @@ Test a single package, but only in a browser
 npm test -- --package @ciscospark/spark-core --browser
 ```
 
+Test a single package, but only in a specific browser
+
+```bash
+USE_BROWSER=chrome npm test -- --package @ciscospark/spark-core --browser
```

Comment:
Is the `BROWSER` flag already used? If not, I'd suggest that since it would match with the widgets project.

Reply:
My main concern was that there would be two "browser"s, one env var for limiting the browser and one command line flag that said to run the browser suites. If you don't think that it will cause confusion, I will change.

Reply:
We could change `--browser` to `--browser-tests` to be more clear if it is an issue. I suspect this would have to change widget-wise too.

Reply:
Ok, I'll make the change to `BROWSER=chrome` and leave `--browser` for now and see how it is received.